distributions/aws/examples/rds-s3/auto-rds-s3-setup.sh

```diff
@@ -0,0 +1,337 @@
+#!/bin/bash
```

Comment:
@AlexandreBrown thanks for adding the python script! Since we now have that in place, would you mind removing the shell script from this PR, we'd like to avoid the duplication.

Reply:
@mbaijal Yes the PR temporarily had both the python script and the bash script while it was WIP (Work-In-Progress). I just completed the re-write in Python of the script. The bash script is no longer present.